lib/rules/sort-prop-types.js

```diff
@@ -57,7 +57,9 @@ module.exports = {
     const propWrapperFunctions = new Set(context.settings.propWrapperFunctions || []);
 
     function getKey(node) {
-      return sourceCode.getText(node.key || node.argument);
+      const key = sourceCode.getText(node.key || node.argument);
+      const hasLeadingQuote = /['"]/.test(key[0]);
```

Comment:
I think this can be changed to:
```js
if (node.key && node.key.value) {
  return node.key.value;
}
return sourceCode.getText(node.key || node.argument);
```